src/components/main-nav/main-nav.pug

```diff
@@ -1 +1,18 @@
-h1= name
+.header__container
+  .header__button
+  a.header__link(href!= logo.link)
+    .header__logo
+      img(src!= logo.img, alt="logo", srcset="")
+  .header__search
+  nav.nav
+    ul.menu
+      each el in links
+        li.menu__item
+          a.menu__link= el.label
+          ul.submenu: each subel in el.links
+            li.submenu__item
+              a.submenu__link(href!= subel.href)= subel.label
+    .header__footer
```

Comment:
Is this really also part of the header? The footer looks like it needs to be a separate component. Could you please explain?